tests/integration/api_v2/http/multisigTransactions.test.js

```diff
@@ -110,5 +110,19 @@ describe('Multisignature Transactions API', () => {
 				.toMap(multisigTransactionSchema, { senderPublicKey: refTransaction.senderPublicKey }));
 			expect(response.meta).toMap(metaSchema);
 		});
+
+		describe('Reject multisignature transactions from the transaction pool', () => {
+			it('Reject a multisignature transaction', async () => {
+				const response = await api.del(`${endpoint}`, {
+					serviceId: refTransaction.serviceId,
+					signatures: inputTransaction.signatures,
+				});
+				expect(response).toMap(goodRequestSchema);
+				expect(response.data).toBeInstanceOf(Array);
+				expect(response.data.length).toBe(1);
+				response.data.forEach(multisigTxn => expect(multisigTxn).toMap(multisigTransactionSchema));
+				expect(response.meta).toMap(metaSchema);
```

Comment:
We do not check for the `rejected` flag within the return response. @MichalTuleja don't we need to map it in the response object?

Reply:
I think there should be a check against it.

Reply:
I mapped the `rejected` flag in the response and updated the corresponding integration test case to verify.